### ISSUE TITLE
Other Basemaps Disappearing 

### ISSUE THREAD

**Issue by egbunse** [StraboSpot/strabo-mobile#435]
@jessicagood @jasonash 
When I loaded 4 separate Mapbox Styles as 4 "Other basemaps" last on the desktop app they worked perfectly, but when I logged back in (this is on the strabo test account) all but 1 were gone. I loaded them again this morning and this keeps happening. Each layer shows up fine on the main map (I can see my data over them and zoom in and out) but when I time out of the site or log out, all but 1 goes missing. Let me know if you need more info.

**Comment by jasonash** [comment]
I suspect this is due to the project not being saved to liveDB when an other basemap is added.

**Comment by egbunse** [comment]
@jasonash it seems projects aren't live saving because a project that I helped someone at the KGS start yesterday is gone today. I was under the impression that projects started on the online app were automatically saved.

**Comment by jasonash** [comment]
@egbunse This is a separate issue. I will open another ticket.

**Comment by jessicagood** [comment]
@jasonash Aren't other maps supposed to be saved to the LiveDB with the project here?https://github.com/StraboSpot/strabo-mobile/blob/master/www/app/maps/other-maps/other-maps.factory.js#L64

**Comment by jasonash** [comment]
@jessicagood Yes, that looks correct to me. I haven't actually duplicated this problem yet, so I'm not sure what is happening. I was just noting what I suspected.

**Comment by egbunse** [comment]
@jasonash with regard to the other basemaps problem, here is the procedure I've been going through: 
I create a Mapbox Style with GeoTiffs; then go to the online Strabo app and go to other basemaps; then I add a few basemaps, verify they're rendering correctly on the main map and then close the browser tab (Chrome). Then when I go back into the project I added them to and only one of the basemaps shows up. I've had this happen on my personal account and the test account.

**Comment by jasonash** [comment]
@jessicagood I found the problem. The modified_timestamp on the project isn't being changed when the call to the liveDB is made. 
Here are two calls made after adding other basemaps:
https://gist.github.com/jasonash/b0f20dd5fe326f4c75e72173b8f625db

**Comment by jessicagood** [comment]
@jasonash That looks like the problem. For other LiveDB project saves `saveProjectItem` is called first which sets the `modified_timestamp` but we don't have access to the `ProjectFactory` in the `OtherMapsFactory` so we'll either have to save the `modified_timestamp` directly or get teh`OtherMapsFactory` working such that it can call the `ProjectFactory`. This would mean storing the data a little differently in the local database but then we wouldn't need the "kludge" (as you called it) to `getThisProject` either. This will take a little more looking at.

**Comment by jasonash** [comment]
Yeah, I think this issue might be an outlier. Maybe we should hold off on trying to get it fixed until we've had time to think about it.
